[2024-12-26T16:27:07.830+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-26T16:27:07.859+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: docker_hello_world.docker_command scheduled__2024-06-29T00:00:00+00:00 [queued]>
[2024-12-26T16:27:07.872+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: docker_hello_world.docker_command scheduled__2024-06-29T00:00:00+00:00 [queued]>
[2024-12-26T16:27:07.872+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2024-12-26T16:27:07.890+0000] {taskinstance.py:2889} INFO - Executing <Task(DockerOperator): docker_command> on 2024-06-29 00:00:00+00:00
[2024-12-26T16:27:07.902+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=450) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-12-26T16:27:07.903+0000] {standard_task_runner.py:72} INFO - Started process 451 to run task
[2024-12-26T16:27:07.906+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'docker_hello_world', 'docker_command', 'scheduled__2024-06-29T00:00:00+00:00', '--job-id', '281', '--raw', '--subdir', 'DAGS_FOLDER/dag_dockeroperator.py', '--cfg-path', '/tmp/tmp66kj5ec1']
[2024-12-26T16:27:07.910+0000] {standard_task_runner.py:105} INFO - Job 281: Subtask docker_command
[2024-12-26T16:27:07.999+0000] {task_command.py:467} INFO - Running <TaskInstance: docker_hello_world.docker_command scheduled__2024-06-29T00:00:00+00:00 [running]> on host 1484ae97611b
[2024-12-26T16:27:08.170+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='docker_hello_world' AIRFLOW_CTX_TASK_ID='docker_command' AIRFLOW_CTX_EXECUTION_DATE='2024-06-29T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-29T00:00:00+00:00'
[2024-12-26T16:27:08.171+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-26T16:27:08.229+0000] {docker.py:379} INFO - Starting docker container from image hello-world
[2024-12-26T16:27:08.235+0000] {docker.py:387} WARNING - Using remote engine or docker-in-docker and mounting temporary volume from host is not supported. Falling back to `mount_tmp_dir=False` mode. You can set `mount_tmp_dir` parameter to False to disable mounting and remove the warning
[2024-12-26T16:27:08.591+0000] {docker.py:73} INFO - 
[2024-12-26T16:27:08.591+0000] {docker.py:73} INFO - Hello from Docker!
[2024-12-26T16:27:08.592+0000] {docker.py:73} INFO - This message shows that your installation appears to be working correctly.
[2024-12-26T16:27:08.592+0000] {docker.py:73} INFO - 
[2024-12-26T16:27:08.593+0000] {docker.py:73} INFO - To generate this message, Docker took the following steps:
[2024-12-26T16:27:08.593+0000] {docker.py:73} INFO -  1. The Docker client contacted the Docker daemon.
[2024-12-26T16:27:08.593+0000] {docker.py:73} INFO -  2. The Docker daemon pulled the "hello-world" image from the Docker Hub.
[2024-12-26T16:27:08.594+0000] {docker.py:73} INFO -     (amd64)
[2024-12-26T16:27:08.594+0000] {docker.py:73} INFO -  3. The Docker daemon created a new container from that image which runs the
[2024-12-26T16:27:08.595+0000] {docker.py:73} INFO -     executable that produces the output you are currently reading.
[2024-12-26T16:27:08.595+0000] {docker.py:73} INFO -  4. The Docker daemon streamed that output to the Docker client, which sent it
[2024-12-26T16:27:08.596+0000] {docker.py:73} INFO -     to your terminal.
[2024-12-26T16:27:08.596+0000] {docker.py:73} INFO - 
[2024-12-26T16:27:08.597+0000] {docker.py:73} INFO - To try something more ambitious, you can run an Ubuntu container with:
[2024-12-26T16:27:08.597+0000] {docker.py:73} INFO -  $ docker run -it ubuntu bash
[2024-12-26T16:27:08.598+0000] {docker.py:73} INFO - 
[2024-12-26T16:27:08.599+0000] {docker.py:73} INFO - Share images, automate workflows, and more with a free Docker ID:
[2024-12-26T16:27:08.599+0000] {docker.py:73} INFO -  https://hub.docker.com/
[2024-12-26T16:27:08.600+0000] {docker.py:73} INFO - 
[2024-12-26T16:27:08.600+0000] {docker.py:73} INFO - For more examples and ideas, visit:
[2024-12-26T16:27:08.601+0000] {docker.py:73} INFO -  https://docs.docker.com/get-started/
[2024-12-26T16:27:08.889+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-26T16:27:08.890+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=docker_hello_world, task_id=docker_command, run_id=scheduled__2024-06-29T00:00:00+00:00, execution_date=20240629T000000, start_date=20241226T162707, end_date=20241226T162708
[2024-12-26T16:27:08.961+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-26T16:27:09.001+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-12-26T16:27:09.003+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
